src/cascadia/TerminalSettingsEditor/MainPage.h

```diff
@@ -5,6 +5,7 @@
 
 #include "MainPage.g.h"
 #include "Utils.h"
+#include "Utils.h"
```

Comment:
wait... we already have Utils.h included...

Reply:
gah probably a bad merge.